### ISSUE TITLE
palette clicks cause the "change" callback to fire

### ISSUE THREAD

**Issue by sp-kilobug** [bgrins/spectrum#321]
In 1.7.0, palette clicks cause the "change" callback to fire. This is probably an issue.
(According to the reference manual, change event "only happens when the input is closed or the 'Choose' button is clicked")

I suggest to replace in the function paletteElementClick(e) :

```javascript
updateOriginalInput(true);
if (opts.hideAfterPaletteSelect) {
    hide();
}
```
by
```javascript
if (opts.hideAfterPaletteSelect) {
    updateOriginalInput(true);
    hide();
}
```

**Comment by Netrilix** [comment]
I like the current behavior of firing a change on every palette select (I use it to update a preview image elsewhere on the page every time a palette color is chosen, even with the palette still open), but I do agree this should be an option, not a default, so it can match the behavior of the color picker.

**Comment by bgrins** [comment]
Thanks for the report - this is fixed by #405
